config/global.ini.php

```diff
@@ -22,6 +22,26 @@
 type = InnoDB
 schema = Mysql
 
+; Database SSL Options START
+; Turn on or off SSL connection to database, possible values for use_ssl: true or false
+use_ssl = false
```

Comment:
Suggest to rename to `enable_ssl` as we tend to use `enable_*` more than `use_*`